get_github_coderabbit_review/coderabbit_fetcher/cache/redis_cache.py

```
@@ -0,0 +1,386 @@
+            self.stats['errors'] += 1
+            return False
+
+        try:
+            # Get all keys with our prefix
+            pattern = f"{self.config.key_prefix}*"
+            keys = self._redis_client.keys(pattern)
+
+            if keys:
+                self._redis_client.delete(*keys)
+
+            return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)